### PR TITLE
full incorporation of the new AntelopeStore.resource into StakeTab and UnstakeTab components

### LINKED ISSUE #662: Auto refresh balances when unstaking
## Description
This issue is a followup to #507. When unstaking (or 'undelegating') successfully, the CPU and NET balances should update automatically. This has been omitted from #507 due to complexities in the code.

### PR DESCRIPTION
**Pull request by Viterbo**
# Fixes #662

## Description
@EJSG pointed out that after the transaction to undelegated resources was successfully executed, the account data didn't refresh automatically and indeed, that was happening. So I took the opportunity to fully incorporate the new `AntelopeStore` prototype into the two components related to the only part implemented for now in the new library. Those components are `StakeTab` and `UnstakeTab`.

This method sendTransaction:
https://github.com/telosnetwork/open-block-explorer/blob/master/src/components/resources/StakeTab.vue#L94-L132

image

Now looks like this, and the best part everything updates Automatically after the action call ends.
https://github.com/telosnetwork/open-block-explorer/blob/206e58c8b49ce3f7d5d8ea8d6709781efd1c1698/src/components/resources/StakeTab.vue#L93-L116

Also the Resource summary and the Balance breakdown were updated and now includes the aggregated amount of resources delegated to other accounts.

image

image

## Test scenarios
Delegate to others should work and update data after success
- go to this link ([testnet](https://deploy-preview-667--obe-testnet.netlify.app/))
- log in with a non-zero TLOS balance account using a real wallet (not cleos)
- go to the wallet tab
   - You should see a new line saying "DELEGATED to others' like the screenshot
- Press the Resource button
   - In the summary, you should see 'DELEGATED TO OTHERS' with the same number
- remember your current state
   - remember the delegated to others number
   - remember your liquid TLOS (shown as available over the inputs)
- go to 'Add CPU/NET' tab
- write in the input a different account name you know
- change to a positive amount of CPU or NET
- press confirm and sign with your wallet
- If success
   - you should receive a popup showing the transfer id (close it)
   - you should see your 'DELEGATED TO OTHERS' number increased
   - you should see liquid TLOS (the available amount) being reduced

Undelegated from others should work and update the date after success.
- go to this link ([testnet](https://deploy-preview-667--obe-testnet.netlify.app/))
- log in with an account that previously had delegated some tokens to another account. Use a real wallet (not cleos)
- go to the wallet tab
   - You should see a new line saying "REFUNDING from stake' like the screenshot
- Press the Resource button
   - In the summary, you should see 'REFUNDING' with the same number
- remember your current state
   - remember the delegated to others number
   - remember the refunding amount
- go to 'Remove CPU/NET' tab
- select an account different from yours
- change to a positive amount of CPU or NET to unstake
- press confirm and sign with your wallet
- If success
   - you should receive a popup showing the transfer id (close it)
   - you should see your 'DELEGATED TO OTHERS' number reduced
   - you should see your funding amount being increased

## Checklist:
-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have cleaned up the code in the areas my change touches
-   [x] My changes generate no new warnings
-   [x] Any dependent changes have been merged and published in downstream modules
-   [x] I have checked my code and corrected any misspellings
-   [x] I have removed any unnecessary console messages

### CODE REVIEW

File: src/api/fuel.ts [hunk 1]
```diff
@@ -435,7 +435,7 @@ function validateActionsContent(
         if (expectedNewActions > 2) {
             validateActionsRamContent(signer, modifiedTransaction);
         }
-        return formatCurrency(totalFee, 4, 'TLOS');
```
Comment (EJSG) [inline src/api/fuel.ts:438]: ouch good catch!

File: src/components/resources/StakeTab.vue [hunk 2]
```diff
@@ -66,7 +64,8 @@ export default defineComponent({
             stakingAccount,
             cpuTokens,
             netTokens,
-            ...mapActions({ sendAction: 'account/sendAction' }),
+            // ...mapActions({ sendAction: 'account/sendAction' }),
```
Comment (EJSG) [inline src/components/resources/StakeTab.vue:67]: rm comment

File: src/components/resources/StakeTab.vue [hunk 3]
```diff
@@ -97,48 +96,24 @@ export default defineComponent({
...
+                to: this.stakingAccount.toLowerCase().trim(),
                 transfer: false,
-                stake_cpu_quantity:
+                cpu_weight:
                     parseFloat(this.cpuTokens) > 0
-                        ? formatCurrency(parseFloat(this.cpuTokens), 4, symbol)
-                        : `0 ${symbol}`,
-                stake_net_quantity:
+                        ? formatCurrency(parseFloat(this.cpuTokens), 4, symbol, true)
+                        : `0.0000 ${symbol}`,
```
Comment (EJSG) [inline src/components/resources/StakeTab.vue:106]: good catch again

File: src/components/resources/UnstakeTab.vue [hunk 4]
```diff
@@ -63,25 +63,26 @@ export default defineComponent({
         const isUnstaking = computed(() => store.resources.isLoading('undelegateResources'));
         const loading = computed(() => store.resources.getLoading());
 
-        const selectOptions = ref<{label:string, value:DelegatedResources}[]>([]);
+        // const selectOptions = ref<{label:string, value:DelegatedResources}[]>([]);
```
Comment (EJSG) [inline src/components/resources/UnstakeTab.vue:66]: remove comment

File: src/components/resources/UnstakeTab.vue [hunk 5]
```diff
@@ -150,12 +166,12 @@ export default defineComponent({
                 transfer: false,
                 cpu_weight:
                     parseFloat(this.cpuTokens) > 0
-                        ? formatCurrency(parseFloat(this.cpuTokens), 4, symbol)
-                        : `0 ${symbol}`,
+                        ? formatCurrency(parseFloat(this.cpuTokens), 4, symbol, true)
```
Comment (EJSG) [inline src/components/resources/UnstakeTab.vue:169]: thanks again

**Review by EJSG — APPROVED** [review]
works great for me :ok_hand:

**Review by donnyquixotic — APPROVED** [review]

**Pull request merged**